Python/specialize.c

```diff
@@ -42,6 +43,31 @@ PyStats *_Py_stats = NULL;
         Py_DECREF(val); \
     } while(0);
 
+void
+_init_pystats(){
+    //Make UOpstats structs for all initial opcodes, with null pointers to deeper opcodes
+    for (int i = 0; i < 512; i++){
+        UOpStats *opcode_stat = _Py_stats_struct.optimization_stats.opcode[i];
+        opcode_stat = PyMem_RawCalloc(1, sizeof(UOpStats));
+        for (int j = 0; j < 512; j++){
+            opcode_stat->next_stats[j] = NULL;
```

Comment:
This probably isn't necessary because `Calloc` is used above (which would already fill the buffer with zeros).